### PR TITLE
Mirror opendatahub-io/kubeflow promoted images to quay.io

### PR DESCRIPTION
**Pull request by samuelvl**

### CODE REVIEW

File: ci-operator/jobs/infra-image-mirroring.yaml [hunk 1]
```diff
@@ -1202,3 +1202,53 @@ periodics:
...
+      - mountPath: /home/mirror/.docker/config.json
+        name: push
+        readOnly: true
+        subPath: config.json
+      - mountPath: /etc/imagemirror
+        name: config
+    volumes:
+    - name: push
+      secret:
+        secretName: registry-push-credentials-quay.io-opendatahub-io
```
Comment (petr-muller) [inline ci-operator/jobs/infra-image-mirroring.yaml:1251]: This secret does not seem to exist

Comment (samuelvl) [reply ci-operator/jobs/infra-image-mirroring.yaml:1251]: May not be synchronized yet? I created before opening the PR in here https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/opendatahub-io/openshift-ci

Comment (samuelvl) [reply ci-operator/jobs/infra-image-mirroring.yaml:1251]: @petr-muller How can I check if the secret is imported? I have no permissions to read that secret in the `app.ci` cluster.

Comment (petr-muller) [reply ci-operator/jobs/infra-image-mirroring.yaml:1251]: Sync is working fine everywhere else, please check that your `secretsync/*` keys contain correct values like instructed in:

https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/#configuring-mirroring-for-new-organization

**Review by petr-muller — APPROVED** [review]

**Pull request merged**